presto-parser/src/test/java/io/prestosql/sql/parser/TestSqlParserErrorHandling.java

```diff
@@ -183,7 +183,7 @@ public void testPossibleExponentialBacktracking2()
                         "OR (f()\n" +
                         "OR (f()\n" +
                         "GROUP BY id",
-                "line 24:1: mismatched input 'GROUP'. Expecting: ')', ',', '.', 'FILTER', 'IGNORE', 'OVER', 'RESPECT', '['");
+                "line 24:1: mismatched input 'GROUP'. Expecting: ')', ',', '.', 'FILTER', 'OVER', '['");
```

Comment:
Is there any reason? It looks like there is no change in the original one.
https://github.com/trinodb/trino/blob/4bcd5b5c35cddd3c076a616282aaf57168236052/core/trino-parser/src/test/java/io/trino/sql/parser/TestSqlParserErrorHandling.java#L195

Reply:
This test fails as we haven't cherry-picked related changes

Reply:
Alright